<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketDialog</class>
 <widget class="QDialog" name="TicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Añadir ticket</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Compañer@</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="concepto_label">
     <property name="text">
      <string>Concepto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="conceptLineEdit">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="Spanish" country="Spain"/>
     </property>
     <property name="maxLength">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Coste</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="costLineEdit"/>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>€ (Usar . para decimales)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Fecha de pago</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="displayFormat">
      <string>d/M/yy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="cancel_Button">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="add_Button">
     <property name="text">
      <string>Añadir</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="addMore_Button">
     <property name="text">
      <string>Añadir más</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="errorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>132</red>
           <green>132</green>
           <blue>132</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="locale">
      <locale language="Spanish" country="Spain"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QCheckBox" name="contributionCheckBox">
     <property name="text">
      <string>Igual contribución</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_Button</sender>
   <signal>clicked()</signal>
   <receiver>TicketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_Button</sender>
   <signal>clicked()</signal>
   <receiver>TicketDialog</receiver>
   <slot>addTicket()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addMore_Button</sender>
   <signal>clicked()</signal>
   <receiver>TicketDialog</receiver>
   <slot>addMoreTickets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contributionCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>TicketDialog</receiver>
   <slot>changeContribution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTicket()</slot>
  <slot>addMoreTickets()</slot>
  <slot>changeContribution()</slot>
 </slots>
</ui>
